bedrock/base/templates/base-new.html

```diff
@@ -0,0 +1,149 @@
+{% from "macros.html" import share_cta with context %}
```

Comment:
As I said though, not a blocker for this PR - just something that might be worth considering, even if only temporary.